app/_hub/kong-inc/key-auth/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Key Authentication
 publisher: Kong Inc.
-version: 2.4.0
+version: 2.2.0
```

Comment:
Put back to the version as I found it but suspect there is a missing doc version